src/uint.rs

```diff
@@ -430,7 +430,7 @@ macro_rules! construct_uint {
 				use core::cmp;
 				use rustc_hex::ToHex;;
 
-				if self.is_zero() { return "0".to_owned(); } // special case.
+				if self.is_zero() { return "0".to_owned(); } // construct_uintspecial case.
```

Comment:
it should be reverted or there should be a space between uint and special

Reply:
this will be reverted. Just a typo.